genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings.go

```diff
@@ -180,39 +179,21 @@ func deletePhoneBaseSettings(ctx context.Context, d *schema.ResourceData, meta i
 			return nil
 		}
 
-		return retry.RetryableError(fmt.Errorf("Phone base settings %s still exists", d.Id()))
+		return retry.RetryableError(fmt.Errorf("phone base settings %s still exists", d.Id()))
 	})
 }
 
 func getAllPhoneBaseSettings(ctx context.Context, sdkConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
 	resources := make(resourceExporter.ResourceIDMetaMap)
+	pp := getPhoneBaseProxy(sdkConfig)
+	phoneBaseSettings, err := pp.getAllPhoneBaseSettings(ctx)
```

Comment:
looks retry logic is removed for this Get API call ?

Reply:
its standard removed in all getAll calls